src/Playwright.Tests/TestServer/SimpleServer.cs

```diff
@@ -179,6 +179,11 @@ public async Task ServeFile(HttpContext context)
         if (!File.Exists(filePath))
         {
             context.Response.StatusCode = StatusCodes.Status404NotFound;
+            context.Response.ContentType = "text/plain";
+            var bytes = Encoding.UTF8.GetBytes($"File not found: {filePath}").AsMemory();
+            await context.Response.BodyWriter.WriteAsync(bytes).ConfigureAwait(false);
+            await context.Response.BodyWriter.CompleteAsync().ConfigureAwait(false);
+            context.Response.ContentLength = bytes.Length;
```

Comment:
```suggestion
            context.Response.ContentType = "text/plain";
            await context.Response.WriteAsync($"File not found: {filePath}").ConfigureAwait(false);
            return;
```

Why does this not work?